README.md

```diff
@@ -91,7 +92,7 @@ const jwt_decode = require('jwt-decode');
 
 #### Include with a script tag
```

Comment:
Note that with the introduction of [import maps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap), it's now possible to load this as an ES module. For example:

```html
<script type="importmap">
  {
    "imports": {
      "jwt-decode": "/node_modules/jwt-decode/build/esm/jwt-decode.js"
    }
  }
</script>
<script type="module">
  import jwtDecode from "jwt-decode";

  const token = "eyJ0eXAiO.../// jwt token";
  const decoded = jwtDecode(token);
</script>
```

Reply:
Thanks, I think that's a bit beyond the scope of our README.